api/api-base/src/test/groovy/com/thoughtworks/go/api/utils/JsonUtil.groovy

```diff
@@ -14,17 +14,18 @@
  * limitations under the License.
  */
 
-package com.thoughtworks.go.api.serializers;
+package com.thoughtworks.go.api.utils
 
-import cd.go.jrepresenter.RequestContext;
-import com.thoughtworks.go.config.CaseInsensitiveString;
+import com.google.gson.JsonElement
+import com.google.gson.JsonObject
+import com.thoughtworks.go.api.util.GsonTransformer
+import groovy.json.JsonOutput
 
-import java.util.function.BiFunction;
+class JsonUtil {
 
-public class CaseInsensitiveStringSerializer implements BiFunction<CaseInsensitiveString, RequestContext, String> {
+  static JsonObject jsonObjectFrom(map) {
+    def jsonString = JsonOutput.toJson(map)
+    GsonTransformer.instance.fromJson(jsonString, JsonElement.class).asJsonObject
```

Comment:
As we've seen before, the behavior of all JSON libraries is subtly different. Can we move this into the GsonTransformer itself?

```java
JsonElement toJsonElement(Object o) {
  return GSON.toJsonTree(o);
}
```